cypher.go

```diff
@@ -14,10 +14,12 @@ import (
 // unmarshalled into it when the query is executed. Result must be a pointer
 // to a slice of structs - e.g. &[]someStruct{}.
 type CypherQuery struct {
-	Statement  string                 `json:"statement"`
-	Parameters map[string]interface{} `json:"parameters"`
-	Result     interface{}            `json:"-"`
-	cr         cypherResult
+	Statement    string                 `json:"statement"`
+	Parameters   map[string]interface{} `json:"parameters"`
+	Result       interface{}            `json:"-"`
+	cr           cypherResult
+	IncludeStats bool `json:"includeStats"`
+	Stats        Stats
```

Comment:
That seems sensible. I've made a start on this.  Tests still to come - I'll see if I can get any time in the coming days.